### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by waterghost-2046**
Provide clear guidance on dependency information and repository configuration to be added to the Maven configuration file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Follow the below steps to add MCProtocolLib as a dependency to your project.
 <repositories>
     <repository>
         <id>opencollab</id>
-        <url>https://repo.opencollab.dev/maven-releases/</url>
+        <url>https://repo.opencollab.dev/maven-snapshots/</url>
```
Comment (onebeastchris) [inline README.md]:
```suggestion
        <url>https://repo.opencollab.dev/main/</url>
```
This'll include both the snapshots and releases repos

File: README.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ Follow the below steps to add MCProtocolLib as a dependency to your project.
 repositories {
     maven { 
         name 'opencollab'
-        url 'https://repo.opencollab.dev/maven-releases/'
+        url 'https://repo.opencollab.dev/maven-snapshots/'
```
Comment (onebeastchris) [inline README.md]:
```suggestion
        url 'https://repo.opencollab.dev/main/'
```

File: README.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ dependencies {
 
 ```kotlin
 repositories {
-    maven("https://repo.opencollab.dev/maven-releases/") {
+    maven("https://repo.opencollab.dev/maven-snapshots/") {
```
Comment (onebeastchris) [inline README.md]:
```suggestion
    maven("https://repo.opencollab.dev/main/") {
```

**Review by onebeastchris — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (waterghost-2046) [reply README.md]: 스크린샷 2024-07-28 오전 10 02 56

**Comment by waterghost-2046** [comment]
https://repo.opencollab.dev/main/ is null

스크린샷 2024-07-28 오전 10 02 56

**Comment by onebeastchris** [comment]
You can't browse `main`, as it just redirects to both `releases` and `snapshots`

**Comment by waterghost-2046** [comment]
> You can't browse `main`, as it just redirects to both `releases` and `snapshots`

I'm sorry, I tested it and confirmed that it works

File: README.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ See the [example](https://github.com/GeyserMC/MCProtocolLib/tree/master/example/
 
 ## Adding as a Dependency
 
-MCProtocolLib builds are published to the [Open Collaboration repository](https://repo.opencollab.dev/#/maven-snapshots/org/geysermc/mcprotocollib/protocol).
+MCProtocolLib builds are published to the [Open Collaboration 'snapshot' repository](https://repo.opencollab.dev/#/maven-snapshots/org/geysermc/mcprotocollib/protocol).
 Follow the below steps to add MCProtocolLib as a dependency to your project.
```
Comment (Konicai) [inline README.md]: If we are going to use `main` in the examples, it wouldn't hurt to mention here that the examples below use `main`, which provides access to both release and snapshot repo.

**Review by onebeastchris — APPROVED** [review]

**Pull request merged**